Sources/MapboxDirections/Lane.swift

```diff
@@ -3,15 +3,15 @@ import Foundation
 /**
  A lane on the road approaching an intersection.
  */
-struct Lane: Equatable {
+public struct Lane: Equatable {
```

Comment:
`Lane` was intentionally removed from the public API. Use `VisualInstruction.Component.lane(indications:isUsable:)` instead.